src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { CardboardClassNamePrefix } from '../../../../../Models/Constants/Constants';
+import { IConditionsListStyles } from './ConditionsList.types';
+
+const classPrefix = CardboardClassNamePrefix + '-conditions-list';
+const classNames = {
+    container: `${classPrefix}-container`
+};
+
+export const getStyles = (): IConditionsListStyles => {
+    return {
+        container: [
+            classNames.container,
+            // 29px is label size, since this is resizing need to calc the 100% - label
+            { overflow: 'auto', height: 'calc(100% - 29px)' }
```

Comment:
Can we round it to 30? We shouldn't have odd numbers in our sizing because it means that something will have uneven spacing on top or bottom.